3515/2/commands/checkin.rb

```diff
@@ -0,0 +1,85 @@
+require_relative '../helper_modules/directory_maker'
+require_relative '../helper_modules/file_write'
+module CheckinCommand
+  def checkin!(*)
+    if user_authorized?
+      checkin_false?
+    else
+      respond_with :message, text: "Извините, но ваш аккаунт не авторизован\nДля авторизации введите
```

Comment:
To keep the source clean I'd recommend to store such strings that are used within user interaction inside YAML files of i18n and get them by key with `I18n.t(key)` method